snooty/rstspec.toml

```diff
@@ -886,7 +886,14 @@ options.type = "string"
 [directive.video]
 help = "Embed a video in your article"
 argument_type = "uri"
-example = """.. video:: ${1: full video url}"""
+options.title = "string"
+options.description = "string"
+options.thumbnail-url = "uri"
+options.upload-date = "iso_8601"
+example = """.. video:: ${1: full video url}
+   :thumbnail-url: ${2: video's thumbnail url}
+   :upload-date: ${3: video's original upload date in ISO-8601 format}
```

Comment:
minor. can we provide examples for iso 8601 for posterity

Reply:
Good idea. I reorganized the example a bit to provide more concrete examples/placeholders. Let me know what you think

Reply:
perfect! 👍